specification/nhs-website-content-api.yaml

```diff
@@ -40,33 +40,44 @@ info:
     Page objects conform to global [schema.org](https://schema.org/) standards.
 
     ## Network access
-    No special network access is required for this API. It is available over the internet.
+    This API is available on the internet and, indirectly, on the [Health and Social Care Network (HSCN)](https://digital.nhs.uk/services/health-and-social-care-network).
+
+    To use this API with [NHS smartcards](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/nhs-smartcards-for-developers), the end user needs an HSCN connection, although internet-facing alternatives to smartcards are available.
+
+    For more details see [Network access for APIs](https://digital.nhs.uk/developer/guides-and-documentation/network-access-for-apis).
```

Comment:
What is this all about?
Health and Social Care Network (HSCN)?

Reply:
It's what people inside health organisations use to access the API. They said that I should include it for people who are used to that